sklearn/impute/_base.py

```diff
@@ -202,6 +206,14 @@ class SimpleImputer(_BaseImputer):
         the missing indicator even if there are missing values at
         transform/test time.
 
+    keep_empty_features : bool, default=False
+        If True, features that consist exclusively of missing values `fit` is
+        called not removed when calling `transform`. The imputed value is
+        always `0` except when `strategy="constant"` for which `fill_value`
+        will be used instead.
```

Comment:
I think there is a small syntax mistake. Also, is using this affirmative phrasing better?
```suggestion
        If True, features that consist exclusively of missing values when
        `fit` is called are returned in results when `transform` is called.
        The imputed value is always `0` except when `strategy="constant"`
        in which case `fill_value` will be used instead.
```